src/graph/test/ideal/append-nodes.ts

```diff
@@ -593,143 +609,120 @@ t.test(
       type: 'prod',
     })
 
-    // Create a breadcrumb-like objects for testing
-    const breadcrumbItem = { name: 'foo', specificity: 1 }
-
-    const completeModifierRefs = new Map([
-      [
-        'foo',
-        {
-          modifier: {
-            type: 'edge' as const,
-            query: '#foo',
-            spec: Spec.parse('bar', '^2.0.0'), // swap with npm:bar@^2.0.0
-            breadcrumb: {
-              first: breadcrumbItem,
-              last: breadcrumbItem,
-              single: true,
-              items: [breadcrumbItem],
-              clear: () => {},
-              clone: function () {
-                return this
-              },
-              comment: '',
-            },
-            value: '^2.0.0',
-            refs: new Set(),
-          },
-          interactiveBreadcrumb: {
-            current: breadcrumbItem,
-            next: () => true,
-            done: true,
-          },
-          originalFrom: graph.mainImporter,
-        },
-      ],
-    ])
```

Comment:
Replaced the "mock everything" approach with a more integrated style for this one that will actually load modifiers from the fixture `vlt.json` file.

Reply:
Yes! This is the right approach.